RCAIDE/Framework/Analyses/Analysis.py

```diff
@@ -0,0 +1,226 @@
+# ----------------------------------------------------------------------------------------------------------------------  
+from RCAIDE.Framework.Core import Container as ContainerBase
+from RCAIDE.Framework.Core import Data
+
+# ----------------------------------------------------------------------------------------------------------------------
+# Analysis
+# ----------------------------------------------------------------------------------------------------------------------   
+class Analysis(Data):
+    """ The top level Analysis class 
+    """
+    def __defaults__(self):
+        """This sets the default values and methods for the analysis.
+        
+            Assumptions:
+                None
+    
+            Source:
+                None
+    
+            Args:
+                None
+    
+            Returns:
+                None 
+            """           
+        self.tag    = 'analysis'
+        self.features = Data()
+        self.settings = Data() 
+        
+    def initialize(self,*args,**kwarg):
+        """Initializes the analysis' specific algorithms.
+        
+            Assumptions:
+                None
+                                            
+            Source:
+                None
+                                            
+            Args:
+                self  : class                           [-] 
+                args  : arguments of the class          [-]
+                kwarg : keyword arguments of the classs [-]
+                                            
+            Returns:
+                None
+            """        
+        return
+    
+    def evaluate(self,*args,**kwarg):
+        """This is used to execute the analysis' specific algorithms.
+        
+            Assumptions:
+                None
+                                            
+            Source:
+                None
+                                            
+            Args:
+                self  : class                           [-] 
+                args  : arguments of the class          [-]
+                kwarg : keyword arguments of the classs [-]
+                                            
+            Returns:
+                None
+                
+            """             
+        raise NotImplementedError 
+    
+    def post_process(self,*args,**kwarg):
+        """Postprocess set of an analysis method
+        
+            Assumptions:
+                None
+                                            
+            Source:
+                None
+                                            
+            Args:
+                self  : class                           [-] 
+                args  : arguments of the class          [-]
+                kwarg : keyword arguments of the classs [-]
+                                            
+            Returns:
+                None
+            """                
+        return 
+    
+    def __call__(self,*args,**kwarg):

```

Comment:
Group NOTE. This one disappears. Keep evaluate as standard within the code

Reply:
Removed. Done